src/rust/ensogl/lib/theme/src/lib.rs

```diff
@@ -204,6 +205,10 @@ define_themes! { [light:0, dark:1]
                 added     = Lcha::green(0.8,1.0), Lcha::green(0.8,1.0);
                 edited    = Lcha::yellow(0.9,1.0), Lcha::yellow(0.9,1.0);
             }
+            error {
+                dataflow.color = Rgba(1.0,0.647,0.0,0.7), Rgba(1.0,0.647,0.0,0.7);
+                panic.color    = Rgba(1.0,0.27,0.0,0.7) , Rgba(1.0,0.27,0.0,0.7);
```

Comment:
We are not using `.color` anywhere else. This is a thee definition, colors are the most basic setting here. We should skip `.color` imo